packages/base/src/Internal/Container.hs

```diff
@@ -205,6 +214,17 @@ optimiseMult = mconcat
 {- | Compute mean vector and covariance matrix of the rows of a matrix.
 
 >>> meanCov $ gaussianSample 666 1000 (fromList[4,5]) (diagl[2,3])
+
+<interactive>:4:52: ERROR:
```

Comment:
Why are these errors here?

Reply:
See above. I have already fixed them, going to push soon.